### PR TITLE
Convert Crossover Options to sheet options

### PR DESCRIPTION
**Pull request by JFranklin3000**
Closes [issue URL]

##### In this PR
- Created App V2 to replace existing Dialog

##### Testing
- validate why app opens at the same size as the character sheet.

### CODE REVIEW

File: templates/app/sheet-options.hbs
Comment (phildominguez) [inline templates/app/sheet-options.hbs:1]: Cleaned up a bit

```hbs
<div class="trait-selector flexcol">
  {{#ifEquals actor.type "npc"}}
    {{#> "systems/essence20/templates/item/parts/sheet-field.hbs" label='E20.IsNPC'}}
    {{#*inline "item-field-inputs"}}
      <input type="checkbox" id="isNPC" name="system.isNPC" {{checked system.isNPC}} />
    {{/inline}}
    {{/"systems/essence20/templates/item/parts/sheet-field.hbs"}}

    {{#> "systems/essence20/templates/item/parts/sheet-field.hbs" label='E20.IsContact'}}
    {{#*inline "item-field-inputs"}}
      <input type="checkbox" id="isContact" name="system.isContact" {{checked system.isContact}} />
    {{/inline}}
    {{/"systems/essence20/templates/item/parts/sheet-field.hbs"}}
  {{/ifEquals}}

  {{#ifNotEquals role.system.version 'powerRanger'}}
    {{#> "systems/essence20/templates/item/parts/sheet-field.hbs" label='E20.CanMorph'}}
    {{#*inline "item-field-inputs"}}
      <input type="checkbox" id="canMorph" name="system.canMorph" {{checked system.canMorph}} />
    {{/inline}}
    {{/"systems/essence20/templates/item/parts/sheet-field.hbs"}}
  {{/ifNotEquals}}

  {{#ifNotEquals role.system.version 'myLittlePony'}}
    {{#> "systems/essence20/templates/item/parts/sheet-field.hbs" label='E20.CanSpellcast'}}
    {{#*inline "item-field-inputs"}}
      <input type="checkbox" id="canSpellcast" name="system.canSpellcast" {{checked system.canSpellcast}}/>
    {{/inline}}
    {{/"systems/essence20/templates/item/parts/sheet-field.hbs"}}
  {{/ifNotEquals}}

  {{#unless altMode}}
    {{#> "systems/essence20/templates/item/parts/sheet-field.hbs" label='E20.CanTransform'}}
    {{#*inline "item-field-inputs"}}
      <input type="checkbox" id="canTransform" name="system.canTransform" {{checked system.canTransform}}/>
    {{/inline}}
    {{/"systems/essence20/templates/item/parts/sheet-field.hbs"}}
  {{/unless}}
</div>
```

File: templates/app/sheet-options.hbs [hunk 1]
```diff
@@ -1,32 +1,45 @@
-<form autocomplete="off" onsubmit="event.preventDefault();">
+<div class="trait-selector">
+  {{#ifEquals actor.type "npc"}}
+    <div class="flexcol">
+      {{#> "systems/essence20/templates/item/parts/sheet-field.hbs" label='E20.IsNPC'}}
+      {{#*inline "item-field-inputs"}}
+        <input type="checkbox" id="isNPC" name="system.isNPC" {{checked system.isNPC}} />
+      {{/inline}}
+      {{/"systems/essence20/templates/item/parts/sheet-field.hbs"}}
```
Comment (phildominguez) [inline templates/app/sheet-options.hbs:8]: I think we should make a new hbs instead of the one we use for item sheets

**Review by phildominguez — CHANGES_REQUESTED** [review]

File: templates/app/sheet-options.hbs
Comment (JFranklin3000) [reply templates/app/sheet-options.hbs:1]: Changed

File: templates/app/sheet-options.hbs (hunk 1 above)
Comment (JFranklin3000) [reply templates/app/sheet-options.hbs:8]: I made a copy for now and put it in templates/app/parts. Should we move the existing hbs files to a subfolder?>

File: templates/app/parts/sheet-field.hbs
Comment (phildominguez) [inline templates/app/parts/sheet-field.hbs:1]: It's not being used in sheet-options.hbs. Also I think just name it field.hbs or something.

**Review by phildominguez — CHANGES_REQUESTED** [review]

File: templates/app/parts/sheet-field.hbs
Comment (JFranklin3000) [reply templates/app/parts/sheet-field.hbs:1]: weird I changed it but it didn't take, I needed to add it to the template.mjs anyway,

File: templates/app/parts/field.hbs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    {{!-- Custom item field edit button goes here --}}
+    {{/item-field-edit}}
+  </div>
+
+  <div class="flexrow item-field-inputs" style="align-items: center;">
+    {{#> item-field-inputs}}
+    {{!-- Custom item inputs go here --}}
+    {{/item-field-inputs}}
+  </div>
+</div>
```
Comment (phildominguez) [inline templates/app/parts/field.hbs:14]:
```suggestion
<div class="flexrow">
  <div class="flexrow" style="gap: 5px;">
    <label style="flex-grow: 0; white-space: nowrap;">{{localize label}}</label>
    {{#> field-edit}}
    {{!-- Custom field edit button goes here --}}
    {{/field-edit}}
  </div>

  <div style="flex-grow: 0;">
    {{#> field-inputs}}
    {{!-- Custom inputs go here --}}
    {{/field-inputs}}
  </div>
</div>
```
The `inline` stuff in sheet-options.hbs will have to be changed from `item-field-edit` to `field-edit` and `item-field-inputs` to `field-inputs`.

**Review by phildominguez — CHANGES_REQUESTED** [review]

**Review by phildominguez — APPROVED** [review]

**Pull request merged**